app/Options.hs

```diff
@@ -52,24 +56,43 @@ optionsInfo = info
 parseCommand :: IO LifeCommand
 parseCommand = execParser optionsInfo
 
+ownerParser :: Parser Owner
+ownerParser = fmap Owner
+     $ strArgument
+     $ metavar "OWNER"
+    <> help "Your github user name"
+
 ----------------------------------------------------------------------------
--- life init
+-- life pull
 ----------------------------------------------------------------------------
 
-data InitOptions = InitOptions
-     { initOptionsOwner :: Owner
-     } deriving (Show)
+data PullOptions = PullOptions
+    { pullOptionsOwner   :: Owner
+    , pullOptionsNoFiles :: [FilePath]
+    , pullOptionsNoDirs  :: [FilePath]
+    } deriving (Show)
+
+pullOptionsParser :: Parser PullOptions
+pullOptionsParser = do
+    pullOptionsOwner <- ownerParser
+
+    -- TODO: reuse LifePath parser here?...
```

Comment:
I think it's better not to do that. The names and description are going to be  different so it makes sense to write it explicitly each time..